.gitignore

```diff
@@ -3,3 +3,5 @@ _site
 .jekyll-cache
 .jekyll-metadata
 vendor
+
+node_modules
```

Comment:
FYI - I removed `node_modules` from version control. You'll want to run `npm install` after you check this out.